### ISSUE TITLE
Invert if-block with continue as only statement in blocks if follow-up code is a single line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#762]
In case a loop contains an `if` block with a negative condition and it's only child is a `continue` then we should report an issue in case the follow-up code is a single statement.

Following example should trigger a violation:
```c#
foreach (var x in something)
{
    if (!x.Whatever)
    {
        continue;
    }
    
    SomeSingleStatementWith(x);
}
```

That code should be fixed into:
```c#
foreach (var x in something)
{
    if (x.Whatever)
    {    
        SomeSingleStatementWith(x);
    }
}
```

The reason is that - by inverting the condition - the fixed code is much easier to read than the original one.

**Comment by RalfKoban** [comment]
Maybe we should also consider the situation that both calls (`if` and other expression) are the last calls inside a block, such as in:
```c#
foreach (var x in something)
{
    DoSomething();

    if (!x.Whatever)
    {
        continue;
    }
    
    SomeSingleStatementWith(x);
}
```

**Comment by RalfKoban** [comment]
In addition to that we should think about inverting the `if` block also in case the condition is positive.

**Comment by RalfKoban** [comment]
For the moment we consider only negative statements because negative logic is much more difficult to understand than positive logic.
